README.md

```diff
@@ -663,7 +663,6 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 ### Web Socket
 * [Fleck](https://github.com/statianzo/Fleck) - Fleck is a WebSocket server implementation in C#. Fleck requires no inheritance, container, or additional references.
 * [SignalR Server](https://github.com/aspnet/signalr) - Real-time web functionality for web apps, including server-side push.
-* [SuperSocket](https://github.com/kerryjiang/SuperSocket) - Light weight, cross platform and extensible socket server application framework.
```

Comment:
I still see they targeting is netstandard here
https://github.com/kerryjiang/SuperSocket/blob/master/src/SuperSocket.Server/SuperSocket.Server.csproj

Reply:
@matteofabbri @ngohungphuc Could we close this PR?